server/routes/daily-attendance-summary/handlers/dailySummary.test.ts

```diff
@@ -124,6 +137,10 @@ describe('Route Handlers - Daily Attendance Summary', () => {
         .calledWith(date, res.locals.user)
         .mockResolvedValue(mockActivities)
 
+      when(await activitiesService.getAllAttendance)
+        .calledWith(date, res.locals.user)
```

Comment:
Does it need the awaits here? Can use .mockResolvedValue(xxxx)?

Reply:
corrected